dashboard/config/environments/test.rb

```diff
@@ -17,9 +17,11 @@
   config.public_file_server.enabled = true
   config.public_file_server.headers = {'Cache-Control' => "public, max-age=3600, s-maxage=1800"}
 
+  is_ci = !!ENV.fetch('CI', nil)
+
   # test environment should use precompiled, minified, digested assets like production,
   # unless it's being used for unit tests.
-  ci_test = !!(ENV['UNIT_TEST'] || ENV.fetch('CI', nil))
+  ci_test = !!(ENV['UNIT_TEST'] || is_ci)
```

Comment:
Having trouble building a mental model of the difference between `is_ci`, `ci_test`, `ENV('UNIT_TEST')` and `ENV.fetch('CI', nil)` Feels like we're solving the same problem several different ways. That or we have a weird venn diagram going on with different scenarios.

I'm not sure how to fill out this table

|   |ENV CI | ENV UNIT_TEST |  is_ci | ci_test |
|---|---|---|---|---|
| local tests | ? | ? | ? | ? |
| drone tests | true | ? | true | true |
| DTT tests | ? | ? | ? | ? |